app/views/admin/forms/index.html.erb

```diff
@@ -23,8 +23,8 @@
 <% end %>
 
 <%- if @forms.present? %>
-<table class="usa-table width-full">
-  <thead class="font-sans-3xs">
+<table class="usa-table usa-table--sticky-header width-full">
+  <thead class="font-sans-3xs z-top">
```

Comment:
`z-top` needed to eliminate bad behavior in the `Actions` column:

<img width="1324" alt="Screenshot 2024-05-25 at 9 14 58 AM" src="https://github.com/GSA/touchpoints/assets/8216592/0b6036f0-e355-4713-aa69-3181c41d8df2">